README.md

```diff
@@ -487,7 +487,47 @@ either by user implicitly calling ```Cancel()``` or when watcher class is destro
   watcher.Cancel();
   etcd.set("/test/key", "43"); /* print response will NOT be called,
                                   since watch is already cancelled */
```

Comment:
A unnecessary curly bracket in the `Watcher Class` section's code sample is removed